corehq/apps/sms/forms.py

```diff
@@ -1200,8 +1200,6 @@ def __init__(self, user: CouchUser, *args, **kwargs):
         for api_id, klass in backend_classes.items():
             if is_superuser_or_contractor(user) or api_id == SQLTelerivetBackend.get_api_id():
                 friendly_name = klass.get_generic_name()
-                if friendly_name == 'Unicel':
-                    continue  # Unicel is a formerly used Gateway which is no longer supported
```

Comment:
nit: This code was added in 97a84c0b75160f509823aa7093210c514ac8becf, and is now being removed in a later commit in the same PR. Ideally this would have been cleaned up using something like `git rebase` to remove all traces of false starts and experiments from the git commit history prior to creating the PR.

However, sometimes these things are only realized after a PR has been created. If that happens, then it may make sense to leave the history as is, or if its a small change and there are not many code comments left by reviewers that would become orphaned, it may make sense to clean up the commit history and force-push the branch. Be sure to leave an explanation on the PR so reviewers are not disoriented by changing commit history.

Feel free to clean up the commit history on this PR if you want to give it a try. I think none of the comments I've made will become difficult to understand if the commits they reference are amended.